lib/chef/knife/ec_backup.rb

```diff
@@ -13,7 +13,11 @@ class EcBackup < Chef::Knife
         require 'chef/chef_fs/config'
         require 'chef/chef_fs/file_system'
         require 'chef/chef_fs/file_pattern'
-        require 'chef/chef_fs/parallelizer'
+        begin
+          require 'chef/chef_fs/parallelizer'
+        rescue LoadError
+          require "chef-utils/parallel_map" unless defined?(ChefUtils::ParallelMap)
```

Comment:
Is it required to add?:

```ruby
using ChefUtils::ParallelMap
```

Reply:
not unless you're actually using the parallel_map Array methods in that particular sourcefile.